core/src/test/scala/org/apache/spark/rdd/CoalescedRDDBenchmark.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.rdd
+import scala.collection.immutable
```

Comment:
Please run `dev/scalastyle` and fix the Scala style issue by adding a new line.
```
$ dev/scalastyle
Scalastyle checks failed at following occurrences:
[error] /Users/dongjoon/PRS/PR-23986/core/src/test/scala/org/apache/spark/rdd/CoalescedRDDBenchmark.scala:19:0: There should at least one a single empty line separating groups java and scala.
[error] Total time: 16 s, completed Mar 8, 2019 11:33:59 AM
```

Reply:
Sorry, somehow IntelliJ didn't highlight that even though I had spark scalastyle set.